ui/Screen/Project/Source.svelte

```diff
@@ -123,5 +144,13 @@
     </div>
   </ActionBar>
 
-  <Router {routes} />
+  {#if activeTab === 'files'}
+    <FilesTab />
+  {:else if activeTab === 'commits'}
+    <CommitsTab />
+  {:else if activeTab === 'commit'}
+    {#if commitHash}
+      <CommitTab {commitHash} />
+    {/if}
+  {/if}
```

Comment:
I think we’re missing some cases here

Reply:
I think it was because the HorizontalMenu had all the tab types in it previously.
Now each screen has its own tab type which lives in the router module.